### DISCUSSION TITLE
Kitchen-Comfyui theme

### DISCUSSION THREAD

**Discussion by kakachiex2** [comfyanonymous/ComfyUI#2328] [Ideas]
I use this theme for Automati1111 and the developer post this beautiful theme for comfyui I think is something that need to reach developer and comfy community to give feedback and support it it's not fully functional but is a great start I hope to see this integrate in comfyui in some way it looks like chaiNNer.
https://github.com/canisminor1990/kitchen-comfyui

**Comment by NeedsMoar** [comment]
I love the RHS nodes menu even if it takes up the space all the time instead of only when the node menu is open... rather use the space than have to scroll up 20 slots if I want a vaedecode or something and only have one hand free because I'm being swarmed by my cats.  :P

**Comment by freeAhao** [comment]
look's pretty good, i like the sidebar and theme,but I prefer the original way of operating in main workspace. Direct connect node to anthoer node look's pretty useful.
